circle.yml

```diff
@@ -23,7 +23,7 @@ dependencies:
 deployment:
   production:
     branch:
-      master
+      sunset
```

Comment:
I want to use this PR for discussion and code review (I might have missed some links/places in get.blog where we reference the search or create new users). But I won't merge with master at the end.